### ISSUE TITLE
When value prop is undefined it goes back to uncontrolled behaviour

### ISSUE THREAD

**Issue by Lyca0n** [TeamWertarbyte/material-ui-chip-input#295]
I believe that a warning should be in place in the case of passing an undefined value prop, in case you do by mistake and you are dealing with your input in a controlled way, it will start ignoring your other event props like onAdd and others.

**Comment by leMaik** [comment]
I agree. 👍 You're not supposed to switch between controlled/uncontrolled. Material-UI also shows a warning if you try to do that.
